src/StartProject.Asp/Startup.cs

```diff
@@ -1,56 +1,38 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-using Moryx;
-using Moryx.AbstractionLayer.Products.Endpoints;
-using Moryx.Asp.Integration;
 using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Mvc.Controllers;
 
 namespace StartProject.Asp
 {
     public class Startup
     {
-        private readonly IApplicationRuntime _moryxRuntime;
-
-        public Startup(IApplicationRuntime moryxRuntime)
-        {
-            _moryxRuntime = moryxRuntime;
-        }
-
-        // ConfigureServices() is called by the host before the Configure() method and will configure the app's
-        // services. By convention, this where configuration options are set, and where services are added the container.
-        // This method is optional for the Startup class.
+        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMoryxFacades(_moryxRuntime);
-
-            services.AddCors(options =>
-            {
-                options.AddPolicy("CorsPolicy", builder => builder
-                .WithOrigins("http://localhost:4200") // Angular app url for testing purposes
-                .AllowAnyMethod()
-                .AllowAnyHeader()
-                .AllowCredentials());
-            });
+            services.AddRazorPages();
 
-            services.AddSignalR();
-            // Add MORYX SignalR hubs
-            services.AddMoryxProductManagementHub();
             services.AddControllers()
-               .AddJsonOptions(jo => jo.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter()));
+                .AddJsonOptions(jo => jo.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter()));
 
             services.AddSwaggerGen(c =>
             {
                 c.CustomOperationIds(api => ((ControllerActionDescriptor)api.ActionDescriptor).MethodInfo.Name);
             });
+
+            services.AddCors(options =>
+            {
+                options.AddPolicy("CorsPolicy", builder => builder
+                    .WithOrigins("http://localhost:8080") // Maintenance.Web usually runs on port 8080
```

Comment:
Why do we host the maintenance on a different port now that it is hosted by Asp? And does this even work, I think the Cors policy must be added before the controllers are 🤔

Reply:
Got caught copying from core :-(